src/User.js

```diff
@@ -72,14 +99,14 @@ class User {
   //     // search savedRecipes for ingredients id
   //     }
   //     }
-  findRecipesByIngredient(savedRecipes, ingredientName) {
+  findRecipesByIngredient(ingredientName) {
```

Comment:
Maybe I am too tired this morning but I dont see this method closing based on brackets?